superset/assets/javascripts/components/ColumnTypeLabel.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  type: PropTypes.string.isRequired,
+};
+
+export default function ColumnTypeLabel({ type }) {
+  let stringIcon;
+  let iconSize = '13';
+  if (type === '' || type === 'expression') {
+    stringIcon = 'ƒ';
+  } else if (type.match(/.*char.*/i) || type.match(/string.*/i) || type.match(/.*text.*/i)) {
+    stringIcon = 'ABC';
+    iconSize = '11';
+  } else if (type.match(/.*int.*/i) || type === 'LONG' || type === 'DOUBLE') {
+    stringIcon = '#';
+  } else if (type.match(/.*bool.*/i)) {
+    stringIcon = 'T/F';
+  } else if (type.match(/.*time.*/i)) {
+    stringIcon = 'time';
+  } else {
+    stringIcon = '?';
```

Comment:
I kinda feel like the null case should be blank rather than a `?`

Reply:
Yeah I agree. I added the ? because there's a specific UNKNOWN type and was thinking maybe having ? as well as a blank could be confusing. But I agree they should either be separate or both blank.